pallets/loans/src/rate_model.rs

```diff
@@ -79,75 +81,93 @@ impl InterestRateModel {
     }
 }
 
+impl InterestRateModelLike for InterestRateModel {
+    fn check_model(&self) -> bool {
+        match self {
+            Self::Curve(curve) => curve.check_model(),
+            Self::Jump(jump) => jump.check_model(),
+            Self::NewCurveName(model) => model.check_model(),
+        }
+    }
+
+    fn get_borrow_rate(&self, utilization: Ratio) -> Option<Rate> {
+        match self {
+            Self::Curve(curve) => curve.get_borrow_rate(utilization),
+            Self::Jump(jump) => jump.get_borrow_rate(utilization),
+            Self::NewCurveName(model) => model.get_borrow_rate(utilization),
+        }
+    }
+}
+
 /// The jump interest rate model
 #[derive(Encode, Decode, Eq, PartialEq, Copy, Clone, RuntimeDebug, Default)]
 pub struct JumpModel {
     /// The base interest rate when utilization rate is 0
     pub base_rate: Rate,
     /// The interest rate on jump utilization point
-    pub jump_rate: Rate,
+    pub max_rate: Rate,
     /// The max interest rate when utilization rate is 100%
-    pub full_rate: Rate,
-    /// The utilization point at which the jump_rate is applied
-    pub jump_utilization: Ratio,
+    pub optimal_rate: Rate,
+    /// The utilization point at which the max_rate is applied
+    pub optimal_utilization: Ratio,
 }
 
 impl JumpModel {
     pub const MAX_BASE_RATE: Rate = Rate::from_inner(100_000_000_000_000_000); // 10%
-    pub const MAX_JUMP_RATE: Rate = Rate::from_inner(300_000_000_000_000_000); // 30%
-    pub const MAX_FULL_RATE: Rate = Rate::from_inner(500_000_000_000_000_000); // 50%
+    pub const MAX_RATE: Rate = Rate::from_inner(300_000_000_000_000_000); // 30%
+    pub const MAX_OPTIMAL_RATE: Rate = Rate::from_inner(500_000_000_000_000_000); // 50%
 
     /// Create a new rate model
     pub fn new_model(
         base_rate: Rate,
-        jump_rate: Rate,
-        full_rate: Rate,
-        jump_utilization: Ratio,
+        max_rate: Rate,
+        optimal_rate: Rate,
```

Comment:
jump_rate should be renamed to optimal_rate.
full_rate should be renamed to max_rate.

Reply:
MAX_RATE and MAX_OPTIMAL_RATE need a corresponding modification.

Reply:
Done